plugin.xml

```diff
@@ -40,9 +40,9 @@
     <dependency id="cordova-plugin-add-swift-support" version="1.7.2">
       <preference name="UseSwiftLanguageVersion" value="3.3" />
     </dependency>
-    <pods-config ios-min-version="9.0" use-frameworks="true"/>
+    <pods-config ios-min-version="9.4" use-frameworks="true"/>
     <source-file src="src/ios/UsabillaCordova.swift" />
     <source-file src="src/ios/FeedbackController.swift" />
-    <pod name="Usabilla" version="5.0.2" />
+    <pod name="Usabilla" version="6.0.2" />
```

Comment:
Wouldn't be an idea to put 6.+ to cover all following minor releases that will not break the public interface anyway?

Reply:
Good thought.  I have removed the version="x.x.x" from the file.
Cordova will then always fetch the latest Usabilla pod